webpack/scenes/RedHatRepositories/components/EnabledRepository/EnabledRepositoryContent.js

```diff
@@ -14,11 +16,17 @@ const EnabledRepositoryContent = ({ loading, disableTooltipId, disableRepository
     >
       <button
         onClick={disableRepository}
-        style={{
+        style={canDisable ? {
           backgroundColor: 'initial',
           border: 'none',
           color: '#0388ce',
-        }}
+        } : {
+          backgroundColor: 'initial',
+          border: 'none',
+          color: '#',
```

Comment:
Is this taking advantage of a default color or something? Maybe better to specify a color..